ecl/eclagent/eclgraph.cpp

```diff
@@ -1198,9 +1161,8 @@ void EclGraph::createFromXGMML(ILoadedDllEntry * dll, IPropertyTree * xgmml, boo
 
 void EclGraph::execute(const byte * parentExtract)
 {
-    GraphRunningState * run = NULL;
     if (agent->queryRemoteWorkunit())
-        run = new GraphRunningState(*this, 0);
+        wu->setGraphState(queryGraphName(), WUGraphRunning);
```

Comment:
Looking at the old code, I don't think it worked before (and probably leaked), but I think the intention was to set the graph state to WUGraphFailed if it failed.  Should this functionality be implemented?